### DISCUSSION TITLE
A strange property of XOR/XNOR

### DISCUSSION THREAD

**Discussion by mbaillif** [logisim-evolution/logisim-evolution#875] [Ideas]
The "Multiple-Input Behavior" property is very strange and it does not correspond to mathematical or logical XOR/XNOR functions.

I propose to remove it.

I have never met an XOR gate with this function and as there are even/odd party elements that solve this function it seems useless on this component.

What do you think ?

**Comment by maehne** [comment]
This has been [discussed in the past](https://sourceforge.net/p/circuit/discussion/479544/thread/e874d0a7/). Carl Burch explained there his reasoning why he implemented this choice. More discussion of the different interpretations for >2-input XOR gates can be found, e.g., [here](https://electronics.stackexchange.com/questions/93713/how-is-an-xor-with-more-than-2-inputs-supposed-to-work) and [here](https://mindhunter74.wordpress.com/2011/04/25/xor-the-interesting-gate/).

**Comment by mbaillif** [comment]
Great, thanks for these links.
And indeed I withdraw my request.

Besides, if I had studied the Venn digram and the wikipedia description better, I would have understood it.
